generators/aws-containers/prompts.js

```diff
@@ -143,13 +170,7 @@ function askCloudFormation() {
             name: 'cloudFormationName',
             message: 'Please enter your stack\'s name. (must be unique within a region)',
             default: this.aws.cloudFormationName || this.baseName,
-            validate: (input) => {
-                if (input) {
-                    return true;
-                }
-
-                return 'Stack\'s name cannot be empty!';
-            }
+            validate: input => ((_.isEmpty(input) || !input.match(CLOUDFORMATION_STACK_NAME)) ? 'Stack name must contain letters, digits, or hyphens ' : true)
```

Comment:
You can ditch the parenthesis. Also I don't know how I feel about the ternary operator for this